driver-examples/tycho/com.datastax.driver.examples.rcp.mailbox.tests/src/main/java/com/datastax/driver/examples/rcp/mailbox/tests/CassandraServer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.examples.rcp.mailbox.tests;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datastax.driver.core.CCMBridge;
+
+public class CassandraServer {
```

Comment:
So I finally managed to have CCM working. The problem is that previously, the test bundle was a fragment of the main bundle (which is the recommended setup for test bundles). But in this situation, the main bundle was activated _before_ the tests.
With my workaround, the test bundle is not a fragment anymore and has its own Activator, which launches a Cassandra instance through CCMBridge right before the main bundle is activated. The downside of this is that the test bundle can now only see public members of the main bundle.
One limitation of the current setup is that the CCM instance is created only once for all tests, so it's not possible to vary the configuration from one test to another (number of nodes, addresses, ports...)